src/containers/PatientProfilePage.tsx

```diff
@@ -43,26 +43,26 @@ const PatientProfilePage = ({
 }) => {
   const history = useHistory();
 
-  const { data, loading, error } = useQuery(
-    mode === "edit" && patientId
+  const { data, loading } = useQuery(
+    mode === 'edit' && patientId
       ? GET_PATIENT_BY_ID(patientId)
       : GET_ALL_PATIENTS
   );
   const patients: Array<Patient> = data ? data.patient : [];
 
   // We need the CCP passed in!
   const [formFields, setFormFields] = useState<FormFields>({
-    barcodeValue: "",
+    barcodeValue: '',
     triage: TriageLevel.GREEN,
-    gender: "Male",
+    gender: 'Male',
     age: null,
-    notes: "",
+    notes: '',
     status: Status.ON_SITE,
     runNumber: null,
   });
 
   useEffect(() => {
```

Comment:
we get an error for this line: `React Hook useEffect has a missing dependency`
not sure if it's important

Reply:
this should be fixed